### PR TITLE
Attempting to fix the terminology illustration. Added line numbers an…

### PR DESCRIPTION
**Pull request by aphillips**
…d additional styling. Fixed the text to reference line numbers and prose descriptions of the examples.

### CODE REVIEW

**Comment by r12a** [comment]
there's some hyphenation appearing in shakespeare_image, which is shall we say a bit problematic! ;)  You'll need to switch hyphenation off for the whole example.

File: index.html [hunk 1]
```diff
@@ -405,48 +405,27 @@ <h3>Terminology and Notation</h3>
...
-            consisting of four empty <code>div</code> elements styled to be
-            orange rectangles). It's also possible that a resource will contain
-            <em>no</em> syntactic content and consist solely of natural language content:
-            for example, a plain text file with a soliloquy from <cite>Hamlet</cite>
-            in it. Notice too that the HTML entity <code>&amp;#x2019;</code>
-            appears in the natural language content and belongs to both the
-          natural language content and the syntactic content in this resource.</li>
+          <li>Everything inside the black rectangle (that is, in this HTML file) is part of the <a>resource</a>.</li>
+          <li><a>Syntactic content</a> in this case includes all of the HTML markup. There are only two strings that are <strong>not</strong> part of the syntactic content: the word <em>"Shakespeare"</em> on line 4 and the sentence <em>"What&#x2019;s in a name? That which we call a rose by any other name would smell as sweet."</em> on line 8. (The HTML entity <q><kbd>&amp;#x2019;</kbd></q> embedded in the sentence on line 8 <em>is</em> part of the syntactic content.)</li>
```
Comment (r12a) [inline index.html:423]: Actually, including the alt attribute value (which you call out below), i think there are 3 string that are not part of the syntactic content, not 2.

File: index.html [hunk 2]
```diff
@@ -405,48 +405,27 @@ <h3>Terminology and Notation</h3>
...
+          <li><a>Natural language content</a> is shown in a <span class="shakespeare">bold blue font with a gray background</span>. In addition to the non-syntactic content, the <kbd>alt</kbd> value on line 7 (<em><q>William Shakespeare</q></em>) contains natural language text.</li>
+          <li>User values are shown in <span class="userValue">italics</span>. In this case there are three user values on line 7: the values of the <kbd>src</kbd>, <kbd>alt</kbd>, and <kbd>id</kbd> attributes of the <kbd>img</kbd> tag. In addition, the value of the <kbd>lang</kbd> attribute on line 1 and the <kbd>charset</kbd> attribute on line 3 are user values.</li>
```
Comment (r12a) [inline index.html:425]: Maybe "User values" -> "User-supplied values"

File: index.html (hunk 1 above)
Comment (aphillips) [reply index.html:423]: The `alt` attribute value is part of the syntactic content. It is also natural language content. The two strings I call out as not being syntactic are the only part of the resource that aren't part of the HTML syntax.

File: index.html (hunk 2 above)
Comment (aphillips) [reply index.html:425]: We could change this. I'll admit that 'user values' isn't super clear. Want to file an issue?

File: index.html (hunk 1 above)
Comment (r12a) [reply index.html:423]: I'm not so sure about that (and never was really convinced).  Sure, the attribute itself is part of the syntax, but whether a string is surrounded by "..." or `<p>...</p>` seems pretty much the same to me.

File: index.html (hunk 2 above)
Comment (r12a) [reply index.html:425]: Done.

**Comment by aphillips** [comment]
Fixed hyphenation in PR bb47226

**Pull request merged**